docs/01_nodeos/03_plugins/chain_plugin/index.md

```diff
@@ -167,6 +167,11 @@ Config Options for eosio::chain_plugin:
   --database-hugepage-path arg          Optional path for database hugepages 
                                         when in "locked" mode (may specify 
                                         multiple times)
+                                        
+  --max-nonprivileged-inline-action-size arg          
+                                        Sets the maximum limit for non-privileged inline actions. 
+                                        The default value is 4 KB and if this threshold is exceeded,
```

Comment:
Not crucial since this is for docs, but it was better to restrict column length to 80 characters like the other entries, especially since this is displayed in a codeblock.

Reply:
Sure. I adjusted the characters. Let me know if it is fine now.